biomaj/bank.py

```diff
@@ -780,19 +810,18 @@ def load_session(self, flow=None, session=None):
                         self.use_last_session = True
 
     def remove_session(self, sid):
-        '''
+        """
         Delete a session from db
 
         :param sid: id of the session
         :type sid: long
         :return: bool
-        '''
+        """
         session_release = None
         _tmpbank = self.banks.find_one({'name': self.name})
         for s in _tmpbank['sessions']:
             if s['id'] == sid:
-                session_release = s['release']
-
+                session_release = self.format_release(s['release'], reverse=True)
```

Comment:
Yes I agree concerning the case with `BmajIndex.remove(self.name, session_release)`.
However, I've made this fix only to `pending` field at first because, it the only case the release is used as a field key in the document. In the other places (`sessions.releease`, `sessions.remoterelease`, `production.release` and `production.remoterelease`) it used as a value, so MongoDB allows values with dot (`.`) in it.
But, as suggested, we could format the release with at the beginning of the bank update.